File: sandbox/php/src/TemplateFilesDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

try {
    $response = (new Dropbox\Sign\Api\TemplateApi(config: $config))->templateFiles(
        template_id: "f57db65d3f933b5316d398057a36176831451a35",
        file_type: null,
    );

    copy($response->getRealPath(), __DIR__ . '/file_response.zip');
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling Template#templateFiles: {$e->getMessage()}";
}

File: sandbox/ruby/src/TemplateFilesDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY";
    # config.access_token = "YOUR_ACCESS_TOKEN";
end

begin
    response = Dropbox::Sign::TemplateApi.new.template_files(
        "f57db65d3f933b5316d398057a36176831451a35",
    )

    FileUtils.cp(response.path, "path/to/file.zip")
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Template#template_files: #{e}"
end
